user/preprocess/input.md

```diff
@@ -116,6 +116,13 @@ The CB-Geo MPM code uses a `JSON` file for input configuration.
       "type": "Linear",
       "xvalues": [0.0, 0.5, 1.0],
       "fxvalues": [0.0, 1.0, 1.0]
+    },
+    {
+      "id": 1,
+      "type": "Linear",
+      "file": "math-function.txt",
+      "xvalues": [],
```

Comment:
If you are setting a file, you shouldn't need x and fx values

Reply:
Got it